lib/portage/dbapi/bintree.py

```diff
@@ -1234,7 +1234,16 @@ def _populate_local(self, reindex=True):
 
         return pkgindex if update_pkgindex else None
 
+    def _run_trust_helper(self):
+        portage_trust_helper = self.settings.get("PORTAGE_TRUST_HELPER")
+        ret = os.system(portage_trust_helper)
+        if ret != 0:
```

Comment:
Let's maybe give a more informative error if it doesn't exist, and tell the user to either install getuto or set PORTAGE_TRUST_HELPER to `true`?

Reply:
Done